src/login/components/LoginFormLedger/LoginFormLedger.js

```diff
@@ -1,34 +1,37 @@
 import React from 'react';
-import { bool, string, func, shape } from 'prop-types';
+import { string, func, shape } from 'prop-types';
 import { noop } from 'lodash';
+import { progressValues } from 'spunky';
 
 import Button from 'shared/components/Forms/Button';
 
 import styles from './LoginFormLedger.scss';
 
 const POLL_FREQUENCY = 1000;
 
+const { FAILED } = progressValues;
+
 const deviceInfoShape = shape({
   manufacturer: string.isRequired,
   product: string.isRequired
 });
 
 export default class LoginFormLedger extends React.Component {
   static propTypes = {
-    disabled: bool,
```

Comment:
We still want to account for the `disabled` prop.  This is used to disable the login button while login is processing.